versions.props

```diff
@@ -13,6 +13,7 @@ com.squareup.okhttp3:okhttp = 4.0.0
 com.uber.nullaway:nullaway = 0.7.9
 io.undertow:undertow-core = 2.0.20.Final
 net.bytebuddy:byte-buddy = 1.10.8
+net.jqwik:* = 1.2.4
```

Comment:
Should we add this to our managed artifacts to get automatic upgrades?

Reply:
yes, PR to update the robots submitted.

Reply:
this is now just sorting all the versions